meta-dts-distro/recipes-dts/dts/dasharo-deploy/dasharo-deploy

```diff
@@ -226,24 +226,26 @@ smbios_migration() {
 }
 
 smmstore_migration() {
-    echo "Beginning SMMSTORE migration process..."
+    echo "Backing up firmware configuration... "
     flashrom -p "$PROGRAMMER_BIOS" ${FLASH_CHIP_SELECT} -r /tmp/dasharo_dump.rom ${FLASHROM_ADD_OPT_READ} --fmap -i FMAP -i SMMSTORE >> $FLASHROM_LOG_FILE 2>> $ERR_LOG_FILE
-    cbfstool /tmp/dasharo_dump.rom read -r SMMSTORE -f /tmp/smmstore.bin
-    echo "Migrate to SMMSTORE section."
-    cbfstool "$BIOS_UPDATE_FILE" write -r SMMSTORE -f /tmp/smmstore.bin -u
+    cbfstool /tmp/dasharo_dump.rom read -r SMMSTORE -f /tmp/smmstore.bin >> $ERR_LOG_FILE 2>&1 || \
+      print_warning "Failed! Default settings will be used."
+    cbfstool "$BIOS_UPDATE_FILE" write -r SMMSTORE -f /tmp/smmstore.bin -u >> $ERR_LOG_FILE 2>&1 || \
+      print_warning "Failed! Default settings will be used."
+    print_green Done.
 }
 
 bootsplash_migration() {
-    echo "Beginning BOOTSPLASH migration process..."
     flashrom -p "$PROGRAMMER_BIOS" ${FLASH_CHIP_SELECT} -r /tmp/dasharo_dump.rom ${FLASHROM_ADD_OPT_READ} --fmap -i FMAP -i BOOTSPLASH >> $FLASHROM_LOG_FILE 2>> $ERR_LOG_FILE
-    cbfstool /tmp/dasharo_dump.rom extract -r BOOTSPLASH -n logo.bmp -f /tmp/logo.bmp >> $ERR_LOG_FILE 2>&1
-    if [ $? -ne 0 ]; then
-      echo "logo.bmp not found, nothing to migrate"
-      return 1
-    fi
-    echo "Migrate logo.bmp to BOOTSPLASH section."
-    cbfstool $BIOS_UPDATE_FILE remove -n logo.bmp -r BOOTSPLASH >> $ERR_LOG_FILE 2>&1
-    cbfstool $BIOS_UPDATE_FILE add -f /tmp/logo.bmp -r BOOTSPLASH -n logo.bmp -t raw -c lzma >> $ERR_LOG_FILE 2>&1
+    # If no custom logo, return from bootsplash_migration early and don't show
+    # unnecessary messages
+    cbfstool /tmp/dasharo_dump.rom extract -r BOOTSPLASH -n logo.bmp -f /tmp/logo.bmp >> $ERR_LOG_FILE 2>&1 || return 1
+    echo "Backing up custom boot logo... "
```

Comment:
```suggestion
    echo -n "Backing up custom boot logo... "
```

Reply:
Added in https://github.com/Dasharo/meta-dts/pull/76/commits/dafd803a5b67db2049ff76ea7a746d62caeb4acc